src/javascript/app/pages/cashier/cashier.js

```diff
@@ -247,13 +244,159 @@ const Cashier = (() => {
                     });
                 }
 
-                if (Currency.isCryptocurrency(currency)) {
+                const fiat_account = Client.hasCurrencyType('fiat');
```

Comment:
Since it's a boolean variable can we rename it to `has_fiat_account` instead?

Reply:
done